tests/test_bs.py

```diff
@@ -392,7 +452,15 @@ def test_client(setup_clients, save_log_on_exit):
     peers = query_message(current_index, testconfig['namespace'], setup_clients.deployment_name, fields, True)
     assert peers == len(setup_clients.pods)
 
-    
+
+def test_add_client(add_client, setup_clients):
+    new_client = add_client()
+    time.sleep(20)
```

Comment:
why is this sleep needed? the client has definitely booted

Reply:
Added comment before